modules/Hoot/managers/api.js

```diff
@@ -754,17 +753,15 @@ export default class API {
                 const dataType = data.inputType === 'Folder' ? 'folder' : 'Dataset';
                 return {
                     message: `'${data.outputname}' ${dataType} Exported`,
-                    status: 200,
                     type: 'success'
                 };
             } )
             .catch( (err) => {
                 console.log( err );
 
                 return {
-                    message: `Failed to export dataset: ${ data.input }`,
-                    status: 500,
-                    type: 'success'
+                    message: `Failed to export dataset: ${ data.outputname }`,
+                    type: 'error'
```

Comment:
These and similar all make the alert message for errors red instead of green.  Also the http status param was never used so removing